test/extended/cluster/cl.go

```diff
@@ -252,7 +251,7 @@ func newProject(nsName string) *projectapi.Project {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: nsName,
 			Annotations: map[string]string{
-				oapi.OpenShiftDisplayName: nsName,
+				"openshift.io/display-name": nsName,
```

Comment:
We have annotations.OpenShiftDisplayName from openshift/api/annotations just for that.